### PR TITLE
feat(#17): add notification on new skill learned

### LINKED ISSUE #17: Display Notification on Training Completion
Currently, the player has to check the skill training screen themselves frequently to notice that training a skill finished and they can select a new skill for training.

To improve user experience and give the player positive feedback on their training, notifications can be used to notify about a training being complete. The [`Notifications` module](https://github.com/Terasology/notifications) provides suitable means to do so.
It [has already been integrated](https://github.com/Terasology/EventualSkills/blob/develop/src/main/java/org/terasology/eventualSkills/systems/EventualSkillsClientSystem.java) with `EventualSkills` to inform the player about the skill training feature at game start.

The [`SkillTrainedEvent`](https://github.com/Terasology/EventualSkills/blob/develop/src/main/java/org/terasology/eventualSkills/events/SkillTrainedEvent.java) is sent by the [`SkillsAuthoritySystem` on completion of the training](https://github.com/Terasology/EventualSkills/blob/develop/src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java#L44). This is the ideal moment to also show a time-limited (10s) notification.

### PR DESCRIPTION
**Pull request by MrGizmo123**
added a popup notification when a new skill is acquired as per #17 

Closes #17

### CODE REVIEW

File: src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java [hunk 1]
```diff
@@ -29,11 +29,32 @@
 import org.terasology.eventualSkills.events.SkillTrainedOwnerEvent;
 import org.terasology.gestalt.assets.ResourceUrn;
 import org.terasology.gestalt.entitysystem.event.ReceiveEvent;
+import org.terasology.notifications.events.ShowNotificationEvent;
+import org.terasology.notifications.model.Notification;
+import org.terasology.engine.logic.players.LocalPlayer;
+import org.terasology.engine.logic.players.event.LocalPlayerInitializedEvent;
```
Comment (jdrueckert) [inline src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java]: Please remove this unused import
```suggestion
```

File: src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java [hunk 2]
```diff
@@ -29,11 +29,32 @@
...
     private static final Logger logger = LoggerFactory.getLogger(SkillsAuthoritySystem.class);
 
+    @In
+    LocalPlayer localPlayer;
+
+    private void showNewSkillNotification(String newSkillName) {
+        Notification notification =
+                new Notification(NOTIFICATION_ID,
+                        "New Skill Acquired",
+                        "Acquired Skill " + newSkillName,
```
Comment (jdrueckert) [inline src/main/java/org/terasology/eventualSkills/systems/SkillsAuthoritySystem.java]: I think we can omit the repetition of "New Skill Acquired" in combination with "Acquired Skill".
How about 
```suggestion
                        "Today I Learned",
                        newSkillName,
```

**Review by jdrueckert — COMMENTED** [review]
First of all: thanks a lot for this PR, tests out like a charm :sparkles: 
Second, is there any particular reason, why you committed your `logback-test.xml`? If not, please remove it from the PR.

**Comment by keturn** [comment]
`module get` supplies `logback-test.xml`, and it's not git-ignored because having it in the repo lets you adjust how things show up on CI. I expect all modules will end up with one. If that sounds like a bad idea, we should talk about that (in some appropriate venue).

**Comment by MrGizmo123** [comment]
This is actually my first PR so I'M kind of new to this. Will remove the XML

**Comment by jdrueckert** [comment]
> `module get` supplies `logback-test.xml`, and it's not git-ignored because having it in the repo lets you adjust how things show up on CI. I expect all modules will end up with one. If that sounds like a bad idea, we should talk about that (in some appropriate venue).

💡 In that case, let's try to get a suitable version of it with the configuration we want in CI in for all omega modules @keturn 
With @skaldarnar's workspace tooling that should be a fairly quick change.

**Comment by jdrueckert** [comment]
> This is actually my first PR so I'M kind of new to this. Will remove the XML

No worries, all good, you're doing great 👍 
Many first-time PRs we get don't even work or implement the desired change, but yours does that and tests out great 😃

**Review by jdrueckert — APPROVED** [review]

**Pull request merged**